ansible/group_vars/all

```diff
@@ -46,27 +55,14 @@ nginx_vhosts:
         auth_basic_user_file /usr/local/biblebox/etc/basicauth;
       }
 
-      location /shared/ {
```

Comment:
I thought the intent of the biblebox itself was that all content could be quickly removed and hidden/destroyed? I'm a little late to the party so I'm sure I missed some discussion.